lib/views/help/requesting.html.erb

```diff
@@ -1,285 +1,298 @@
-        that you can only request "environmental information" and tells you what that
-        means. It is quite broad.
-      </p>
-
-      <p>You can, of course, request environmental information from other
-        authorities. Just make a Freedom of Information (FOI) request as normal. The
-        authority has a duty to work out if the Environmental Information Regulations
-        (EIR) is the more appropriate legislation to reply under.
-      </p>
-    </dd>
-
-    <dt id="multiple">Can I make the same to request to lots of authorities, e.g. all councils? <a href="#multiple">#</a> </dt>
-
-    <dd>We ask you to first send a test version of your request to a few
-      authorities. Their responses will help you improve the wording of your request,
-      so that you get the best information when you send the request to all of
-      the authorities. There is currently no automated system for sending the request
-      to the other authorities, you must copy and paste it by hand.
-
-    </dd>
-
-    <dt id="offsite">I made a request off the site, how do I upload it to the archive?<a href="#offsite">#</a> </dt>
-
-    <dd><%= site_name %> is an archive of requests made through the site,
-      and does not try to be an archive of all FOI requests. We'll never support uploading
-      other requests. For one thing, we wouldn't be able to verify that other
-      responses actually came from the authority. If this really matters to you,
-      you can always make the same request again via <%= site_name %>.
-    </dd>
-
-    <dt id="moderation">How do you moderate request annotations? <a href="#moderation">#</a> </dt>
-
-    <dd>
-      <p>Annotations on <%= site_name %> are to help
-        people get the information they want, or to give them pointers to places they
-        can go to help them act on it. We reserve the right to remove anything else.
-      </p>
-      <p>Endless, political discussions are not allowed.
-        Post a link to a suitable forum or campaign site elsewhere.</p>
-      <dd>
-
-  </dl>
-
-  <p><strong>Next</strong>, read about <a href="<%= help_privacy_path %>">your privacy</a> --&gt;
-
-    <div id="hash_link_padding"></div>
-</div>
\ No newline at end of file
+<div id="left_column_flip">
+
+<h1 id="making_requests"><%= @title %> <!-- <a href="#making_requests">#</a> --> </h1>
+<dl>
+
+<dt id="do_i_have_the_right">Do I have the right to make a request?</dt>
+<dd>
+<p>Every citizen has a right of access to information and records in the possession of public bodies. The right applies to
+all information and records of Government ministries, departments, local governments, statutory corporations and bodies,
+commissions and other Government organs and agencies, unless specifically exempted by law.
+</p>
+</dd>
+
+
+
+<dt id="which_authority">I'm not sure which authority to make my request to, how can I find out? <a href="#which_authority">#</a> </dt>
```

Comment:
Should the `<a href="#which_authority">#</a>` be commented out? It looks odd to have just one link on the page